_pages/programme/calendar.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Calendar
+permalink: /programme/calendar/
+sidebar:
+  nav:  programme
+classes: wide
+toc: true
+toc_sticky: true
+fullcalendar: true
+---
+
+<div id='calendar'></div>
+
+<script>
+
+ document.addEventListener('DOMContentLoaded', function() {
+   var calendarEl = document.getElementById('calendar');
+
+   var calendar = new FullCalendar.Calendar(calendarEl, {
+     headerToolbar: {
+       left: 'prev,next today',
+       center: 'title',
+       right: 'dayGridMonth,listWeek,listDay'
+     },
+     buttonText: {
+       week: 'week',
+       day: 'day'
+     },
+     timeZone: 'UTC',
+     initialView: 'dayGridMonth',
+     editable: true,
+     selectable: true,
+     events: [{% for event in site.events %}{
+       title  : '{{ event.title }}',
+       url    : '{{ event.url }}',
+       allDay : {% if event.all_day %}true{% else %}false{% endif %},{% if event.end_date %}end    : '{{ event.end_date | date_to_xmlschema }}'{% endif %}
```

Comment:
```suggestion
       allDay : {% if event.all_day %}true{% else %}false{% endif %},
       {% if event.end_date %}end    : '{{ event.end_date | date_to_xmlschema }}'{% endif %}
```
just for readability

Reply:
Are you sure? That will render an empty / blank line for most entries.

Reply:
hmm, to me the readability of the code is more important than how it renders. But if you like, we can also do

```suggestion
       allDay : {% if event.all_day %}true{% else %}false{% endif %},
       {% if event.end_date %}end    : '{{ event.end_date | date_to_xmlschema }}'{% else %}//{% endif %}
```
which puts two `//` instead of a blank line

Or maybe even

```suggestion
       allDay : {% if event.all_day %}true{% else %}false{% endif %},
       {% unless event.end_date %}// {% endunless %}end    : '{{ event.end_date | date_to_xmlschema }}'
```